doc_src/help.txt

```diff
@@ -13,6 +13,8 @@ If a `SECTION` is specified, the help for that command is shown.
 
 If the BROWSER environment variable is set, it will be used to display the documentation. Otherwise, fish will search for a suitable browser.
 
+If you prefer to use a different browser (other than as described above) for fish help, you can set the fish_help_browser environment variable. This variable may be set as an array, where the first element is the browser command and the rest are browser options.
```

Comment:
s/environment variable/variable/ as there is nothing that requires this to be an environment variable.